### ISSUE TITLE
Works but console shows broken message

### ISSUE THREAD

**Issue by SillyBirds** [reactiflux/discord-irc#492]
Hello,
I had the bot working fine before, but for other reasons I had to reinstall the droplet it's on, so I installed the bot again. The bot works fine on irc and discord, but for some reason it shows broken lines in console:
```
2019-06-12T19:23:52.195Z info: Connected to Discord { message: undefined,
  level: undefined,
  timestamp: '2019-06-12T19:23:52.195Z',
  splat: undefined,
  [Symbol(level)]: 'info' }
2019-06-12T19:23:53.304Z info: Connected to IRC { message: undefined,
  level: undefined,
  timestamp: '2019-06-12T19:23:53.304Z',
  splat: undefined,
  [Symbol(level)]: 'info' }
```
(I'm using the same config file from my previous installation)
Was there something extra I didn't install?

**Comment by Throne3d** [comment]
I think these messages are normal as of #410 and #453, though they're probably an unintended side effect.

From a developer perspective, it looks like parameters we send in `logger.info` that _weren't_ the first parameter are now being placed into a "splat" property on the logged object, instead of being appended to the message. Additionally, this and the timestamp property being non-empty, _and_ the fact that util.inspect will serialize undefined values (so `{message: undefined}` doesn't become `"{}"`) mean that our old check (seeing if the stringified version is `{}`) fails and we print the whole thing.

I tried looking into Winston to fix it, but it seems to make some awkward assumptions that I'm having trouble sorting out – it says you should use `format.splat()` since the splat functionality was removed, but _that_ itself relies on the splat objects being _objects_, not strings (since it just merges it with the logged object it creates), and trying to replicate `console.log` behavior is probably doable but a little ugly (special-case strings, otherwise use `util.inspect`?) – additionally, sometimes the logged object uses `level` as a property, and sometimes `Symbol(level)`. I guess I could fix it, but there are so many awkward assumptions going on that I feel we must have violated something somewhere.

**Comment by Throne3d** [comment]
Fixed in #554!
